release-please-config.json

```diff
@@ -0,0 +1,14 @@
+{
+  "bootstrap-sha": "95df901b104dadc8902117c87d2e412d78d6a620",
```

Comment:
Is this a valid sha?

Reply:
No, I admit I stole this from another repo.. I tested that the process worked in my forked repo
I've just tried to run
```
release-please bootstrap \
  --token=$GITHUB_TOKEN \
  --repo-url=skyerus/open-feature-operator \
  --release-type=go
```
to generate the sha but I'm getting a 404. I might need to ask for some help on this later

Reply:
Just do a `git log` and grab a sha. Since haven't used conventional commits in the past, any commit sha should work.

Reply:
Speaking of conventional commits, you should add the commit linter here. We'll always want to update the merging options to use the PR title as the commit message.

Reply:
Done